public/js/fxa-analytics.js

```diff
@@ -126,7 +126,6 @@ function sendRecommendationPings(ctaSelector) {
       hitCallback: function() {
         removeUtmsFromUrl();
       },
-      nonInteraction: true,
```

Comment:
Because the Growth team was having issues tracking the experiment in GA, we want to minimize changes at this time. I propose we re-open #1643 to track that issue in the future.